assets/sass/components/header.scss

```diff
@@ -1,17 +1,72 @@
 .header {
+  position: relative;
   align-items: center;
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 1.2em;
-  padding: 0.6em 2%;
+  margin-bottom: 4em;
+  padding: 0.6em 2% 0.6em 2%;
+  border-bottom: 8px solid $color-primary;
+  transition-duration: 0.3s;
+  transition-property: all;
 
-  &__language-select {
+  @media only screen and (min-width: 712px) {
+    padding: 0.6em 110px 0.6em 2%;
+  }
```

Comment:
This might also be taken care of by the `responsive-container()`. If we use that for all elements, the widths of all blocks will always be aligned.